components/micro-gateway-cli/src/main/resources/templates/listeners.mustache

```diff
@@ -23,22 +23,22 @@ http:ListenerConfiguration secureServiceEndpointConfiguration = { {{>http2}},
 {{>docker}}
 {{>secureKubernetesIngress}}
 {{>kubernetesServiceHttps}}
-listener gateway:APIGatewaySecureListener apiSecureListener = new(gateway:DEFAULT_HTTPS_PORT, secureServiceEndpointConfiguration);
+listener gateway:APIGatewaySecureListener apiSecureListener = new(9095, secureServiceEndpointConfiguration);
```

Comment:
Any reason for not using default constants?

Reply:
Kubernetes annotations are read at the compile time. So it mandatory to have the port value during the compile time